rtnetlink/Cargo.toml

```diff
@@ -22,6 +22,8 @@ thiserror = "1"
 netlink-packet-route = { path = "../netlink-packet-route", version = "0.6" }
 netlink-proto = { path = "../netlink-proto", version = "0.5" }
 byteordered = "0.5.0"
+nix = "0.19.0"
+tokio = { version = "0.2.6", features = ["full"] }
```

Comment:
Sorry for missing this earlier, but as far as I can see we only need `tokio::task`, so we probably shouldn't pull in the entire tokio crate.

Reply:
I'll do a patch and fix this

Reply:
Patch is here: https://github.com/little-dude/netlink/pull/132